src/main/java/org/folio/rest/impl/PermsAPI.java

```diff
@@ -1896,7 +1896,6 @@ private static Permission getRealPermObject(PermissionUpload entity) {
       subPerms.add(s);
     }
     perm.setSubPermissions(subPerms);
-    perm.setMutable(entity.getMutable());
     perm.setVisible(entity.getVisible());
```

Comment:
When can `mutable` be true then? Maybe `perm.setMutable(true)` here since these are user defined perms?

Reply:
Yes. Must be set to true here and a unit test should test this this indeed the case. Eg you could add to `RestVerticleTest.testPostPermsPermissions`:

```java
   response = send(HttpMethod.GET, "/perms/permissions/" + uuid, null, context);
   context.assertEquals(response.code, 200);
   context.assertTrue(Boolean.TRUE.equals(response.body.getBoolean("mutable")), response.body.encodePrettily());
```

Reply:
`mutable` isn't a required field, so I figured we'd just omit it, but I'm fine with making it explicit.  I'll fix this in a moment.

Reply:
In any case you're right that a unit test should be updated/added.